app/products/calls/components/calls_custom_message/calls_custom_message.tsx

```diff
@@ -0,0 +1,186 @@
+    return {
+        messageStyle: {
+            flexDirection: 'row',
+            color: changeOpacity(theme.centerChannelColor, 0.6),
+            fontSize: 15,
+            lineHeight: 20,
+            paddingTop: 5,
+            paddingBottom: 5,
+        },
+        messageText: {
+            flex: 1,
+        },
+        joinCallIcon: {
+            padding: 12,
+            backgroundColor: '#339970',
+            borderRadius: 8,
+            marginRight: 5,
+            color: 'white',
+            overflow: 'hidden',
+        },
+        phoneHangupIcon: {
+            padding: 12,
+            backgroundColor: changeOpacity(theme.centerChannelColor, 0.6),
+            borderRadius: 8,
+            marginRight: 5,
+            color: 'white',
+            overflow: 'hidden',
+        },
+        joinCallButtonText: {
+            color: 'white',
+        },
+        joinCallButtonIcon: {
+            color: 'white',
+            marginRight: 5,
+        },
+        startedText: {
+            color: theme.centerChannelColor,
+            fontWeight: 'bold',
+        },
+        joinCallButton: {
+            flexDirection: 'row',
+            padding: 12,
+            backgroundColor: '#339970',
+            borderRadius: 8,
+            alignItems: 'center',
+            alignContent: 'center',
+        },
+        timeText: {
+            color: theme.centerChannelColor,
+        },
+        endCallInfo: {
+            flexDirection: 'row',
+            alignItems: 'center',
+            alignContent: 'center',
+        },
+        separator: {
+            color: theme.centerChannelColor,
+            marginLeft: 5,
+            marginRight: 5,
+        },
+    };
+});
+
+export const CallsCustomMessage = ({
+    post, currentUser, author, isMilitaryTime, teammateNameDisplay,
+    currentCallChannelId, leaveChannelName, joinChannelName,
+}: Props) => {
+    const intl = useIntl();
+    const theme = useTheme();
+    const style = getStyleSheet(theme);
+    const serverUrl = useServerUrl();
+    const timezone = getUserTimezone(currentUser);
+
+    const confirmToJoin = Boolean(currentCallChannelId && currentCallChannelId !== post.channelId);
+    const alreadyInTheCall = Boolean(currentCallChannelId && currentCallChannelId === post.channelId);
+
+    const joinHandler = () => {
+        if (alreadyInTheCall) {
+            return;
+        }
+
+        leaveAndJoinWithAlert(intl, serverUrl, post.channelId, leaveChannelName || '', joinChannelName || '', confirmToJoin, joinCall);
+    };
+
+    if (post.props.end_at) {
+        return (
+            <View style={style.messageStyle}>
+                <CompassIcon
+                    name='phone-hangup'
+                    size={16}
+                    style={style.phoneHangupIcon}
+                />
+                <View style={style.messageText}>
+                    <Text style={style.startedText}>{'Call ended'}</Text>
```

Comment:
Yep, that's a good point. v1 wasn't translatable since we were alpha. I've added a ticket to do a complete look over: https://mattermost.atlassian.net/browse/MM-45945
All future UI will be translatable (e.g. https://github.com/mattermost/mattermost-mobile/pull/6502 )

Reply:
yes, let's make this all of this translatable, I think it was fine for v1 but for v2 we should address this already.